Mathlib/Analysis/NormedSpace/LinearIsometry.lean

```diff
@@ -566,16 +565,16 @@ instance : EquivLike (E ≃ₛₗᵢ[σ₁₂] E₂) E E₂ where
   left_inv e := e.left_inv
   right_inv e := e.right_inv
 
-instance : SemilinearIsometryEquivClass (E ≃ₛₗᵢ[σ₁₂] E₂) σ₁₂ E E₂ where
+instance instSemilinearIsometryEquivClass :
+    SemilinearIsometryEquivClass (E ≃ₛₗᵢ[σ₁₂] E₂) σ₁₂ E E₂ where
   map_add f := map_add f.toLinearEquiv
   map_smulₛₗ e := map_smulₛₗ e.toLinearEquiv
   norm_map e := e.norm_map'
 
+-- TODO: Shouldn't these `CoeFun` instances be scrapped?
```

Comment:
cc @Vierkantor

Reply:
Too late now but yes, this instance should be unnecessary. (It probably won't do too much harm if we can guarantee that the definition is exactly `⟨DFunLike.coe⟩`.)